src/clj/harja/palvelin/integraatiot/velho/varusteet.clj

```diff
@@ -478,6 +478,7 @@
                                 :otsikot otsikot
                                 :url (str varuste-api-juuri-url "/hakupalvelu/api/v1/haku/kohdeluokat")}
                 urakka-velho-oid (q-urakat/hae-urakan-velho-oid db {:id urakka-id})
+                _ (assert urakka-velho-oid "Urakalle ei löytynyt vastaavaa Velho-oidia.")
```

Comment:
Voisko tarkistuksen tehdä ilman asserttia? 
Tässä on keskustelu, jonka perusteella olisin sanomassa, että assertin käytössä kannattaa olla hirmu varovainen: https://clojureverse.org/t/pragmatic-programmer-should-we-all-use-assert/9106/7

Reply:
Muutin tuon perinteisempään malliin! Hyvä hoksautus! Luin kyllä että assert heittää Errorin mutta ajattelin että tuo tulee devatessa vastaan tuo virhe. Ehkä se on kuitenkin parempi muuttaa jos tuo tulisikin käyttäjällä vastaan.